packages/docs-app/src/examples/core-examples/common/iconSelect.tsx

```diff
@@ -77,9 +80,6 @@ export class IconSelect extends React.PureComponent<IconSelectProps> {
     };
 
     private filterIconName = (query: string, iconName: IconName | typeof NONE) => {
-        if (iconName === NONE) {
-            return true;
-        }
         if (query === "") {
```

Comment:
this prevents loading the full list on empty query - there's over 600 icons and it's a bit laggy if showing all... and doesn't quite feel worth bringing in a virtualized list for

Reply:
I was a bit surprised it was starting to struggle with just 600 items, in case this raises any concerns